test/ui-test/src/drivers/SquishDriver.py

```diff
@@ -14,20 +14,36 @@
 import squish
 import object
 import names
+import test
+
 
 # The default maximum timeout to find ui object
-_MAX_WAIT_OBJ_TIMEOUT = 5000 #[milliseconds]
+_MAX_WAIT_OBJ_TIMEOUT = 5000
+
+# The default minimum timeout to find ui object
+_MIN_WAIT_OBJ_TIMEOUT = 500
+
 
 # Waits for the given object is loaded, visible and enabled.
 # It returns a tuple: True in case it is found. Otherwise, false. And the object itself.
-def is_loaded_visible_and_enabled(objName, timeout = _MAX_WAIT_OBJ_TIMEOUT):
+def is_loaded_visible_and_enabled(objName, timeout=_MAX_WAIT_OBJ_TIMEOUT):
 	obj = None
 	try:
 		obj = squish.waitForObject(getattr(names, objName), timeout)
 		return True, obj
 	except LookupError:
 		return False, obj
 
+	
+def verify_screen_is_loaded(objName, timeout=_MAX_WAIT_OBJ_TIMEOUT):
```

Comment:
Should it return true/false? 
Just to be consistent, if we use `verify_XX` what should we expect? I.e: in `verify_text` method it has a retrun statement but here and in `verify_object_not_enabled`, no.